src/index.ts

```diff
@@ -1,12 +1,22 @@
 import { MochaInstanceOptions } from "mocha";
+import Plus from '@/assets/plus.svg';
+import { Button } from '@/components/common/Button/Button';
+import type { SwapAssetExt } from '@/root/trade/_components/TradeView/types';
+import { PREVIEW_TYPES } from '@/services/contexts/preview/constants';
+import { useItemPreviewContext } from '@/services/contexts/preview/ItemPreviewContext';
+import type { SwapAsset } from '@/services/hooks/use-filter-swaps/types';
+import styles from './trade-view-item.module.scss';
```

Comment:
how about separate file and test file for frontend configs? :)

Reply:
makes sense